SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -2226,11 +2227,17 @@ FRPCErrorInfo USpatialReceiver::ApplyRPCInternal(UObject* TargetObject, UFunctio
 					Trace_SpanId CauseSpanId = EventTracer->GetSpanId(Id);
 					TOptional<Trace_SpanId> SpanId = EventTracer->CreateSpan(&CauseSpanId, 1);
 					EventTracer->TraceEvent(FSpatialTraceEventBuilder::CreateProcessRPC(TargetObject, Function), SpanId);
+					EventTracer->SpanIdStack.AddToLayer(SpanId.GetValue());
 				}
 			}
 
 			TargetObject->ProcessEvent(Function, Parms);
 
+			if (bUseEventTracer)
```

Comment:
There could be an unmatched pop if RPCId == 0 here https://github.com/spatialos/UnrealGDK/pull/2625/files#diff-31d5d9cf515637825ada963684e29f7c6af68f298d27b7dbccf084010b5fcdb5R2220